pages/en/about/previous-releases.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Releases
+layout: download-releases.hbs
+modules: '<code>NODE_MODULE_VERSION</code> refers to the ABI (application binary interface) version number of Node.js, used to determine which versions of Node.js compiled C++ add-on binaries can be loaded in to without needing to be re-compiled. It used to be stored as hex value in earlier versions, but is now represented as an integer.'
+---
+
+# Releases
+
+Major Node.js versions enter _Current_ release status for six months, which gives library authors time to add support for them.
+After six months, odd-numbered releases (9, 11, etc.) become unsupported, and even-numbered releases (10, 12, etc.) move to _Active LTS_ status and are ready for general use.
+_LTS_ release status is "long-term support", which typically guarantees that critical bugs will be fixed for a total of 30 months.
+Production applications should only use _Active LTS_ or _Maintenance LTS_ releases.
+
+![Releases](https://raw.githubusercontent.com/nodejs/Release/main/schedule.svg?sanitize=true)
+
+### Looking for latest release of a version branch?
```

Comment:
We should remove this line or add component In `MDX` Component in question :
[more info about how this nodejs page had render](https://github.com/nodejs/nodejs.dev/blob/main/src/pages/download/releases.tsx)

<img width="1016" alt="Capture d’écran 2023-10-19 à 12 31 03" src="https://github.com/nodejs/nodejs.org/assets/97875033/df95a278-91c5-4618-a5c6-3367ef36e3f2">

Reply:
> We should remove this line or add component In MDX Component in question :
[more info about how this nodejs page had render](https://github.com/nodejs/nodejs.dev/blob/main/src/pages/download/releases.tsx?rgh-link-date=2023-10-19T10%3A31%3A31Z)

The component is there, lol.

Reply:
wait on my first check of preview it's doesn't appear. strange...